src/App.js

```diff
@@ -1,98 +0,0 @@
-import React from "react";
-import { StrictMode } from "react";
-import ReactDOM from "react-dom";
-import { useState } from "react";
-import Header from "./Header";
-import Question from "./Question";
-import Suggestions from "./Suggestions";
-
-const App = () => {
-    const [quizStarted, setQuizStarted] = useState(false);
-    const quizQuestions = ["How are you feeling today?*", "Do you prefer indoors or outdoor?*", "Would you like to socialize or plan solo?*"];
-    const quizOptions = [["Relaxed", "Cheery", "Snoozy", "Productive"], ["Outdoors", "Indoors"], ["Socialize", "Solo"]];
-    const images = ["https://images.pexels.com/photos/3807758/pexels-photo-3807758.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260", "https://cdn.ramseysolutions.net/media/article-images/budget-date-night-l.jpg", "https://hips.hearstapps.com/clv.h-cdn.co/assets/18/02/1515470256-levi-guzman-268866.jpg"];
-    const [mood, setMood] = useState("");
-    const [location, setLocation] = useState("");
-    const [socialize, setSocialize] = useState("");
-
-    const [firstQuesAnswered, setFirstQuesAnswered] = useState(false);
```

Comment:
hmm - I like that the parent component is managing the state for the children. Though we could use a different data structure manage this information.

```
import question from quizQuestions.json;


const [activeQuestion, updateActiveQuestion] = useState(0)
const [questionDetails, updateQuestionDetails] = useState(question)

handleQuestionDetailUpdate = () => {
  // this should use the activeQuestionIndex to determine which object in the array of question should be updated
  // identify what kind of information should be passed to this function to update the question details in a specific manner
}
```

Reply:
I  see the intent of using the above data structure, however, not sure hot to implement the changes but will do some research. If I have doubts, I'll ping you :)